src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2081,6 +2087,48 @@ private bool RefMustBeObeyed(bool isDelegateCreation, ArgumentSyntax argumentSyn
             return hadError;
         }
 
+        private BoundExpression BindArgumentValue(DiagnosticBag diagnostics, ArgumentSyntax argumentSyntax, bool allowArglist, RefKind refKind)
+        {
+            if (argumentSyntax.Expression != null)
+            {
+                return BindArgumentExpression(diagnostics, argumentSyntax.Expression, refKind, allowArglist);
+            }
+
+            var typeSyntax = argumentSyntax.Type;
+
+            bool isConst = false;
+            bool isVar;
+            AliasSymbol alias;
```

Comment:
We are discarding the alias here. Please test that GetTypeInfo on the type expression of an out var declaration still preserves the alias, i.e. when calling SemanticMode.GetAliasInfo()?